int32 base_deg
int32 waist_deg
int32 shoulder_deg